### PR TITLE
Remove GDPR Modal

### PR DESCRIPTION
**Pull request by johnwcallahan**
## Description

This removes the GDPR Modal that appeared on every page load and route change. Policy acceptance is now handled by [login.linode.com](login.linode.com).

Note: if the API returns a policy account notification, it will still be displayed in the notifications menu. Clicking the notification takes you to [login.linode.com/policies](login.linode.com/policies).

### CODE REVIEW

File: packages/manager/src/features/TopMenu/UserNotificationsMenu/UserNotificationsList.tsx [hunk 1]
```diff
@@ -126,19 +126,6 @@ const createClickHandlerForNotification = (
...
-    notification.type === 'notice' &&
-    notification.label === `We've updated our policies.`
-  ) {
-    return (e: React.MouseEvent<HTMLElement>) => {
-      window.location.href = `https://manager.linode.com/account/policy`;
-    };
-  }
-
   const type = path<string>(['entity', 'type'], notification);
   const id = path<number>(['entity', 'id'], notification);
```
Comment (acourdavault) [inline packages/manager/src/features/TopMenu/UserNotificationsMenu/UserNotificationsList.tsx]: Here Path could be replace by `?` pattern

File: packages/manager/src/features/TopMenu/UserNotificationsMenu/UserNotificationsMenu.tsx [hunk 2]
```diff
@@ -196,7 +175,7 @@ class UserNotificationsMenu extends React.Component<CombinedProps, State> {
     this.setState({ anchorEl: undefined });
 }
 
-const isPrivacyPolicityNotification = (n: Notification) =>
+const isPrivacyPolicyNotification = (n: Notification) =>
   n.type === `notice` && n.label === `We've updated our policies.`;
```
Comment (acourdavault) [inline packages/manager/src/features/TopMenu/UserNotificationsMenu/UserNotificationsMenu.tsx]: Here while this works, I feel like in terms of design the fat that we rely on the *exact* label to decide if this is a policy change is a little risky.
What if someone updated in API the test to `We have...`?
Then our message is broken

**Comment by acourdavault** [comment]
Also in this function in UserNotificationsMenu, we could use an `if` but this is just a tiny refactor
```
const userAgentDetection = () => {
  switch (b.name) {
    case 'ie':
      return (
        <Typography>
          Your Web Browser (<strong>{b.name}</strong>) is not compatible with
          the Linode Manager. Please update to{' '}
          <a
            href="https://www.microsoft.com/en-us/windows/microsoft-edge"
            target="_blank"
            aria-describedby="external-site"
            rel="noopener noreferrer"
          >
            Microsoft Edge
          </a>{' '}
          for more security, speed and the best experience on this site.
        </Typography>
      );
    default:
      return undefined;
  }
};
```

File: packages/manager/src/features/TopMenu/UserNotificationsMenu/UserNotificationsMenu.tsx (hunk 2 above)
Comment (johnwcallahan) [reply packages/manager/src/features/TopMenu/UserNotificationsMenu/UserNotificationsMenu.tsx]: Agreed, but API will likely be removing this notification altogether soon.

**Pull request merged**